src/useEditor.ts

```diff
@@ -230,10 +212,7 @@ const useEditor = (
     doCommand,
     replaceState,
     // advanced use:
```

Comment:
No need in that comment since "replaceView"is removed

Reply:
Removed.